packages/storybook/src/components/transactions-table/types.ts

```diff
@@ -0,0 +1,109 @@
+// Standalone types for Transactions Table - recreated without GraphQL dependencies
+
+export const Currency = {
+  Ils: 'ILS',
+  Usd: 'USD',
+  Eur: 'EUR',
+  Gbp: 'GBP',
+  Jpy: 'JPY',
+  Btc: 'BTC',
+  Eth: 'ETH',
+} as const;
+
+export type Currency = typeof Currency[keyof typeof Currency];
+
+export const AccountType = {
+  BankAccount: 'BankAccount',
+  CreditCard: 'CreditCard',
+  Cash: 'Cash',
+  CryptoWallet: 'CryptoWallet',
+  Investment: 'Investment',
+  Loan: 'Loan',
+  Other: 'Other',
+} as const;
+
+export type AccountType = typeof AccountType[keyof typeof AccountType];
+
+export interface Amount {
+  raw: number;
+  formatted: string;
+}
```

Comment:
Better import enums and types directly from the client app, this duplication may cause issues in the future and be headache to maintain